metadata/base/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-deployment
+  labels:
+      deployment: metadata
```

Comment:
please use `commonLabels` in kustomization.yaml  to replace all labels/matchLabels

Reply:
Added it. Thanks for the review!